solutions/usaco-568.mdx

```diff
@@ -13,49 +13,128 @@ author: Jesse Choe
 ## Implementation
 
 ```cpp
-// Created by Jesse Choe - Bronze Template
-
 #include <bits/stdc++.h>
 using namespace std;
 
-void setIO(string s){ // FastIO See General -> Input/Output
-	ios_base::sync_with_stdio(0); cin.tie(0);
-	freopen((s+".in").c_str(),"r",stdin);
-	freopen((s+".out").c_str(),"w",stdout);
-}
+const int LEN = 100;
+
+int main() {
+	freopen("speeding.in", "r", stdin);
+	freopen("speeding.out", "w", stdout);
 
-int main(){
-	setIO("speeding");
-	int N, M; cin >> N >> M;
+	int N, M;
+	cin >> N >> M;
 
 	int start = 0;
-	int limit[100];
-	for(int i = 0; i < N; i++){
-		int length, speed; cin >> length >> speed;
-		for(int j = start; j < start + length; j++){
+	vector<int> limit(LEN);
+	for (int i = 0; i < N; i++) {
+		int length, speed;
+		cin >> length >> speed;
+		for (int j = start; j < start + length; j++) {
 			limit[j] = speed;
 		}
 		start += length;
 	}
 
 	start = 0;
-	int bessie[100];
-	for(int i = 0; i < M; i++){
-		int length, speed; cin >> length >> speed;
-		for(int j = start; j < start + length; j++){
+	vector<int> bessie(LEN);
+	for (int i = 0; i < M; i++) {
+		int length, speed;
+		cin >> length >> speed;
+		for (int j = start; j < start + length; j++) {
 			bessie[j] = speed;
 		}
 		start += length;
 	}
 
-	int ans = 0;
-	for(int i = 0; i < 100; i++){
-		ans = max(ans, bessie[i]-limit[i]);
+	int worst = 0;
+	for (int i = 0; i < LEN; i++) {
+		worst = max(worst, bessie[i] - limit[i]);
 	}
-	cout << ans << endl;
+	cout << worst << endl;
 }
-
 ```
 
 </CPPSection>
+<JavaSection>
+
+```java
+import java.io.*;
```

Comment:
idt this is necessary since the official sol is java

Reply:
what's the harm in adding an alternative implementation? other than a minor increase in file size

Reply:
well it's the same idea as the offical sol, so it's repetitive